subsys/fs/ext2/ext2_diskops.c

```diff
@@ -669,6 +669,9 @@ int64_t ext2_inode_remove_blocks(struct ext2_inode *inode, uint32_t first)
 
 	max_lvl = get_level_offsets(inode->i_fs, first, offsets);
 
+	if (max_lvl < 0)
```

Comment:
curly braces

Reply:
someday I'll remember. Today is not that day.